index.html

```diff
@@ -6,12 +6,15 @@
 		
     <title>MG | Portfolio</title>
     <link rel="stylesheet" type="text/css" href="Normalize.css">
-	
     <link rel="stylesheet" type="text/css" href="style.css">
+	<link href="https://fonts.googleapis.com/css?family=Space+Mono&display=swap" rel="stylesheet">
```

Comment:
best to include font files before your stylesheet (where you will reference them)